packages/py-api/py_api/controllers/teams_controller.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from bson.errors import InvalidId
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
+
+
+class TeamsController:
+
+    def fetch_teams() -> JSONResponse:
+        teams = list(t_col.find())
```

Comment:
Why do you need the list constructor here?

Reply:
Otherwise the result is this ![image](https://github.com/AUBGTheHUB/monolith/assets/38375079/0375271b-b40a-4efa-90d6-d8fad6f94e37)